docs/whatsnew/3.0.rst

```diff
@@ -25,6 +35,54 @@ smaller improvements and bug fixes, which are described in the
 * xxx pull requests have been merged since v2.0
 * xxx distinct people have contributed code
 
+.. _whatsnew-3.0-velocities-skycoord-gaia:
+
+Full support for velocities in ``astropy.coordinates``
+======================================================
+
+
+.. _whatsnew-3.0-constants-contextmanager:
+
+Context managers for versioned ``astropy.constants``
+====================================================
+
+
+.. _whatsnew-3.0-showtable:
+
+``showtable`` command line script
+=================================
+
+
+.. _whatsnew-3.0-hdf5-mixins:
+
+HDF5 Table writer supports mix-in columns (e.g. time, coordinates, etc.)
+========================================================================
+
+
+.. _whatsnew-3.0-fits-time-support:
+
+Support for FITS time standard
+==============================
+
+
+.. _whatsnew-3.0-fits-performance:
+
+FITS table performance enhancements
+===================================
+
+
+.. _whatsnew-3.0-kuiper-functions:
+
+Kuiper functions added to ``astropy.stats``
+===========================================
+
+
+.. _whatsnew-3.0-pytest-plugins:
+
+Astropy pytest plugins moved to external packages
```

Comment:
@eteq , now that I read this file as a whole, I feel like this section about `pytest` has already been covered under "Renamed/removed functionality". Would seem repetitive to mention this in two different sections?

Reply:
Oh, duh, good point @pllim!  I think it should be highlighted in the bullet points but you're right it should just point to the existing section instead of being a separate one.  Will make a PR to do that.